sklearn/preprocessing/data.py

```diff
@@ -46,6 +46,10 @@
     'minmax_scale',
 ]
 
+DEPRECATION_MSG_1D = "Passing 1d arrays as data is deprecated and "
+"will be removed in 0.18. Reshape your data either using"
+"X.reshape(-1, 1) if your data has a single feature or"
+"X.reshape(1, -1) if it contains a single sample."
```

Comment:
This does not do what you think it does: only the first line is in the message. Instead use enclosing parens:

```python
DEPRECATION_MSG_1D = ("Passing 1d arrays as data is deprecated and "
                      "will be removed in 0.18. Reshape your data either "
                      "using X.reshape(-1, 1) if your data has a single "
                      "feature or X.reshape(1, -1) if it contains a single "
                      "sample.")
```